app/code/Magento/SendFriend/Model/SendFriend.php

```diff
@@ -372,6 +372,20 @@ public function setSender($sender)
         return $this->setData('_sender', new \Magento\Framework\DataObject($sender));
     }
 
+    /**
+     * Set the reply to email address
+     *
+     * @param array $sender
+     * @return void
+     */
+    public function setReplyTo($sender)
+    {
+        $this->_transportBuilder->setReplyTo(
```

Comment:
If this is coming from the general website user, I think we should validate the email format and escape name field before setting it to Reply-To.